system-tests/test_forwarding.py

```diff
@@ -113,3 +113,170 @@ def test_forwarder_sends_pv_updates_single_pv_string(docker_compose):
     assert union_string.Value() == stop_command
 
     cons.close()
+
+
+def test_forwarder_sends_pv_updates_single_pv_long(docker_compose):
+    """
+    Test the forwarder pushes new PV value when the value is updated.
+
+    NOTE: longs are converted to ints in the forwarder as they will fit in a 32 bit integer
+    :param docker_compose: Test fixture
+    :return: none
+    """
+
+    data_topic = "TEST_forwarderData_long_pv_update"
+    pvs = [PVLONG]
+
+    prod = ProducerWrapper("localhost:9092", CONFIG_TOPIC, data_topic)
+    prod.add_config(pvs)
+    # Wait for config to be pushed
+    sleep(2)
+
+    cons = create_consumer()
+
+    # Update value
+    change_pv_value(PVLONG, 5)
+    # Wait for PV to be updated
+    sleep(5)
+    cons.subscribe([data_topic])
+
+    first_msg = poll_for_valid_message(cons).value()
+    log_data_first = LogData.LogData.GetRootAsLogData(first_msg, 0)
+    check_message_pv_name_and_value_type(log_data_first, Value.Value.Int, PVLONG)
+    check_int_value_and_equality(log_data_first, 0)
+
+    second_msg = poll_for_valid_message(cons).value()
+    log_data_second = LogData.LogData.GetRootAsLogData(second_msg, 0)
+    check_message_pv_name_and_value_type(log_data_second, Value.Value.Int, PVLONG)
+    check_int_value_and_equality(log_data_second, 5)
+    cons.close()
+
+
+def test_forwarder_sends_pv_updates_single_pv_enum(docker_compose):
+    """
+    Test the forwarder pushes new PV value when the value is updated.
+
+    NOTE: longs are converted to ints in the forwarder as they will fit in a 32 bit integer
+    :param docker_compose: Test fixture
+    :return: none
+    """
+
+    data_topic = "TEST_forwarderData_enum_pv_update"
+    pvs = [PVENUM]
+
+    prod = ProducerWrapper("localhost:9092", CONFIG_TOPIC, data_topic)
+    prod.add_config(pvs)
+    # Wait for config to be pushed
+    sleep(2)
+
+    cons = create_consumer()
+
+    # Update value
+    change_pv_value(PVENUM, "START")
+    # Wait for PV to be updated
+    sleep(5)
+    cons.subscribe([data_topic])
+
+    first_msg = poll_for_valid_message(cons).value()
+    log_data_first = LogData.LogData.GetRootAsLogData(first_msg, 0)
+    check_message_pv_name_and_value_type(log_data_first, Value.Value.Int, PVENUM)
+    check_int_value_and_equality(log_data_first, 0)
+
+    second_msg = poll_for_valid_message(cons).value()
+    log_data_second = LogData.LogData.GetRootAsLogData(second_msg, 0)
+    check_message_pv_name_and_value_type(log_data_second, Value.Value.Int, PVENUM)
+    check_int_value_and_equality(log_data_second, 1)
+    cons.close()
+
+
+def test_forwarder_updates_multiple_pv_same_type(docker_compose):
```

Comment:
I don't think there is enough value in this test to justify the extra run time, given that you also have `test_forwarder_updates_multiple_pv_different_type`